helm/sealed-secrets/values.yaml

```diff
@@ -56,6 +56,12 @@ secretName: "sealed-secrets-key"
 ## @param updateStatus Specifies whether the Sealed Secrets controller should update the status subresource
 ##
 updateStatus: true
+## @param skipRecreate Specifies whether the Sealed Secrets controller should skip recreating removed secrets
+## Setting it to false allows to optionally restore backward compatibility in low priviledge
```

Comment:
Just noticed this small typo ^^
This message is talking about the case in which `skipRecreate` is actually set to `true`

Reply:
Fixed per https://github.com/bitnami-labs/sealed-secrets/pull/1371